apps/openmw/CMakeLists.txt

```diff
@@ -140,18 +140,50 @@ target_link_libraries(openmw
 )
 
 if (ANDROID)
+    set (OSG_PLUGINS
+        -Wl,--whole-archive
+        ${OSG_PLUGINS_DIR}/libosgdb_dds.a
+        ${OSG_PLUGINS_DIR}/libosgdb_bmp.a
+        ${OSG_PLUGINS_DIR}/libosgdb_tga.a
+        ${OSG_PLUGINS_DIR}/libosgdb_gif.a
+        ${OSG_PLUGINS_DIR}/libosgdb_jpeg.a
+        ${OSG_PLUGINS_DIR}/libosgdb_png.a
+        -Wl,--no-whole-archive
+    )
     target_link_libraries(openmw
         EGL
         android
         log
         dl
         MyGUIEngineStatic
-        cpufeatures
         BulletCollision
         LinearMath
+        z
+        osg
+        osgDB
+        osgAnimation
+        osgText
+        osgUtil
+        osgShadow
```

Comment:
Better to use e.g. ${OSGSHADOW_LIBRARIES} instead of plain 'osgShadow'.

Also, we don't use osgShadow yet.

Also, osg, osgUtil, osgText, osgDB and osgAnimation are included in ${OPENSCENEGRAPH_LIBRARIES} so shouldn't need to specify them separately.